src/viur/toolkit/db.py

```diff
@@ -61,8 +61,8 @@ def txn(_key: db.Key, _name: str, _value: float | int, _start: float | int) -> f
 
 
 def set_status(
-    key: _KeyType,
-    values: dict | t.Callable[[skeleton.SkeletonInstance | db.Entity], None],
+    key: t.Optional[_KeyType] = None,
+    values: dict | t.Callable[[skeleton.SkeletonInstance | db.Entity], None] | None = None,
```

Comment:
This makes no sense to me, a default value which isn't allowed
```suggestion
    values: dict | t.Callable[[skeleton.SkeletonInstance | db.Entity], None],
```